lib/alertwildfire_API.py

```diff
@@ -62,8 +62,13 @@ def get_individual_camera_info(cameraID):
     listofCameras = response.json()
     return listofCameras[0]
 
-def request_current_image(outputDir, cameraID, timeStamp = None):
-    if not timeStamp:
+def request_current_image(outputDir, cameraID):
+    camera_info = get_individual_camera_info(cameraID)
+    if camera_info["image"]["time"]:
+        timeStamp = camera["image"]["time"]###need to convert their format
+    elif camera_info["position"]["time"]:
+        timeStamp = camera["position"]["time"]###need to convert their format
+    else:
```

Comment:
I like the new approach of moving the time functionality here.  But, let's follow Mark's guidance and go one step further by changing the invokeApi with a fetch of camera['image']['url'].  You'll need to refactor a bit -- perhaps add an optional url parameter to invokeApi.  It's fine if you prefer to defer that to a separate next code change.